website.hits

```diff
@@ -1,7 +1,7 @@
 #!/bin/sh -eu
 
 readonly CURSOR_FILE=data/websites-cursor
-readonly STATS_IP=`host -t A stats.c4dt.org | cut -d ' ' -f 4`
+readonly STATS_IP=`host -t A stats.c4dt.org | tail -n1 | cut -d ' ' -f 4`
```

Comment:
Something from here seems to be more robust, should we change back to an `A` entry later:

https://unix.stackexchange.com/questions/20784/how-can-i-resolve-a-hostname-to-an-ip-address-in-a-bash-script

My favorite is `dit +short`.

Reply:
This PR makes it "`A`-change resitant", as the last line of `host` is the recursively resolved IPv4.

Using `dig` simplifies the line, and both come from the same pkg, so yeah, using `dig +short`.

Reply:
OK, now I understand. I parsed the `tail` wrongly as jumping a number of lines at the beginning... But that would be `tail -n -1`, or so. Or `head -n -1`? I don't remember.

Reply:
Haha, I'm always having issue remembering which is which.
* `head -n -1` ends before the last line
* `tail -n +2` starts at the second line